Wikipedia/Code/WMFAccountCreationViewController.swift

```diff
@@ -226,11 +223,14 @@ class WMFAccountCreationViewController: WMFScrollViewController, WMFCaptchaViewC
                 self.dismiss(animated: true, completion: {
                     presenter?.wmf_showEnableReadingListSyncPanel(theme: self.theme, oncePerLogin: true)
                 })
-        }, failure: { error in
-            self.setViewControllerUserInteraction(enabled: true)
-            self.enableProgressiveButtonIfNecessary()
-            WMFAlertManager.sharedInstance.showErrorAlert(error as NSError, sticky: true, dismissPreviousAlerts: true, tapCallBack: nil)
-        })
+            case .failure(let error):
+                self.setViewControllerUserInteraction(enabled: true)
+                self.enableProgressiveButtonIfNecessary()
+                WMFAlertManager.sharedInstance.showErrorAlert(error as NSError, sticky: true, dismissPreviousAlerts: true, tapCallBack: nil)
+            default:
+                break
```

Comment:
would it be of any utility to add an assert for the default case here? or do we not care if result is ever `alreadyLoggedIn` here?

Reply:
ah gotcha. would it be useful to assert in case someone changes that in the future?